README.md

```diff
@@ -18,6 +18,11 @@ Web Components are a new browser feature that provides a standard component mode
 - CSS changes
     - The CSS WG works on [CSS Scoping](https://drafts.csswg.org/css-scoping/) and [CSS Shadow Parts](https://drafts.csswg.org/css-shadow-parts/), which help dealing with shadow trees with various selectors. Various other parts of CSS and its object model are also impacted by shadow trees and directly worked on in the various CSS specificaions.
     - [Issues against the CSS WG repository](https://github.com/w3c/csswg-drafts/labels/topic%3A%20shadow).
+- HTML Modules
+    - Successor to the abandoned *[HTML Imports](https://w3c.github.io/webcomponents/spec/imports/)*, allows HTML markup to be requested by a component. Spec work is being incubated upstream in whatwg/html (see [PR](todo)). **Note: PR creation is still in progress; replace with actual URL when opened.** For new issues, please file against whatwg/html or here in w3c/webcomponents. See also the [HTML Modules explainer](proposals/html-modules-explainer.md), [initial proposal](proposals/html-modules-proposal.md), and [earlier design ideas](proposals/HTML-Imports-and-ES-Modules.md).  
+    - [Issues raised in the HTML Standard](https://github.com/search?q=org%3Awhatwg+label%3A%22topic%3A+html%20modules%22+is%3Aopen). **Note: that label doesn't exist yet; get exact label when it's created**
```

Comment:
Same.

Reply:
Dropped the link for the `topic: html modules` label. If you'd like to make such a label, that'd be great--I can add a search query here for it. If you'd rather wait, that's fine to :)

Reply:
Label created!

Reply:
Woot! Nice. Link added.